plugins/webhooks/index.js

```diff
@@ -385,6 +394,15 @@ async function createPREvents(options, request) {
                 chainPR
             });
 
+            let subscribedEvent = false;
```

Comment:
don't you need similar changed in `createEvents` function ? This function is called only for PR events

Reply:
yeah they are coming up as further additions. This was just a work in progress.